tests/integration-tests/src/flows/faucet/initFaucet.ts

```diff
@@ -0,0 +1,45 @@
+import { FlowProps } from '../../Flow'
+import { extendDebug } from '../../Debugger'
+import { FixtureRunner } from '../../Fixture'
+import { BuyMembershipHappyCaseFixture, TransferInvitesHappyCaseFixture } from '../../fixtures/membership'
+import { SetBudgetFixture } from '../../fixtures/workingGroups/SetBudgetFixture'
+import _ from 'lodash'
+import BN from 'bn.js'
+
+export default async function initFaucet({ api, env, query }: FlowProps): Promise<void> {
+  const debug = extendDebug('flow:initFaucet')
+  debug('Started')
+
+  // Get membership working group leader
+  const membershipWorkingGroup = 'membershipWorkingGroup'
+  const [, membershipLeader] = await api.getLeader(membershipWorkingGroup)
+
+  // Grant lead invitation quota
+  const proposalDetails = api.createType('ProposalDetails', { SetMembershipLeadInvitationQuota: 50 })
+
+  // The membership working group should have a budget allocated
+  const budgets: BN[] = [new BN(1000000)]
+  const setGroupBudgetFixture = new SetBudgetFixture(api, query, membershipWorkingGroup, budgets)
+  await new FixtureRunner(setGroupBudgetFixture).runWithQueryNodeChecks()
+
+  // Create a membership account for faucet
+  const faucetAccounts = (await api.createKeyPairs(1)).map(({ key }) => key.address)
+  const happyCaseFixture = new BuyMembershipHappyCaseFixture(api, query, faucetAccounts)
+  await new FixtureRunner(happyCaseFixture).runWithQueryNodeChecks()
+  const [faucetMemberId] = happyCaseFixture.getCreatedMembers()
+
+  // Give the faucet member accounts some funds (they need some funds for extrinsics)
+  await api.treasuryTransferBalanceToAccounts(faucetAccounts, new BN(200))
+
+  // Use the above membershipLeader to give faucet account permission to invite other members
+  // const transferInvitesHappyCaseFixture = new TransferInvitesHappyCaseFixture(
+  //   api,
+  //   query,
+  //   { memberId: membershipLeader.member_id, account: membershipLeader.role_account_id.toString() },
+  //   { memberId: faucetMemberId, account: faucetAccounts[0] },
+  //   10
+  // )
+  // await new FixtureRunner(transferInvitesHappyCaseFixture).runWithQueryNodeChecks()
```

Comment:
This is currently failing with the below mentioned error:

```
Error: InvitesTransferred event details not found in result: {"dispatchError":{"Module":{"index":"20","error":"9"}},"dispatchInfo":{"weight":"614,596,000","class":"Normal","paysFee":"Yes"},"events":[{"phase":{"ApplyExtrinsic":"2"},"event":{"method":"ExtrinsicFailed","section":"system","index":"0x0001","data":[{"Module":{"index":"20","error":"9"}},{"weight":"614,596,000","class":"Normal","paysFee":"Yes"}]},"topics":[]}],"status":{"InBlock":"0x9e4348bf4a773bd99a56354bbc4ccff41447cf78ac290c89fa8a5ae8a0598b7c"}}
    at Api.retrieveMembershipEventDetails (/Users/anuj/Joystream/joystream/tests/integration-tests/src/Api.ts:466:13)
    at TransferInvitesHappyCaseFixture.execute (/Users/anuj/Joystream/joystream/tests/integration-tests/src/fixtures/membership/TransferInvitesHappyCaseFixture.ts:64:18)
    at TransferInvitesHappyCaseFixture.runner (/Users/anuj/Joystream/joystream/tests/integration-tests/src/Fixture.ts:24:5)
    at FixtureRunner.run (/Users/anuj/Joystream/joystream/tests/integration-tests/src/Fixture.ts:179:5)
    at FixtureRunner.runWithQueryNodeChecks (/Users/anuj/Joystream/joystream/tests/integration-tests/src/Fixture.ts:202:5)
    at initFaucet (/Users/anuj/Joystream/joystream/tests/integration-tests/src/flows/faucet/initFaucet.ts:42:3)
    at /Users/anuj/Joystream/joystream/tests/integration-tests/src/Job.ts:99:11
```

<img width="1439" alt="Screenshot 2022-02-11 at 5 39 27 PM" src="https://user-images.githubusercontent.com/7795956/153589126-e3a1cecc-a002-4554-acd8-67413e08170b.png">

Reply:
This is failing most likely because the lead doesn't have any invites to transfer, and the extrinsic failed, instead of producing the expected event.

Reply:
cc: @Lezek123 I think it would be a really nice addition for the fixtures to grab the error name when we fail to find the event we are looking for?